src/data/pokemon/species_info/gen_3_families.h

```diff
@@ -11433,7 +11436,7 @@ const struct SpeciesInfo gSpeciesInfoGen3[] =
         .frontAnimId = ANIM_GROW_VIBRATE,
         .backPic = gMonBackPic_DeoxysNormal,
         .backPicSize = MON_COORDS_SIZE(64, 56),
-        .backPicYOffset = 6,
+        .backPicYOffset = P_GBA_STYLE_SPECIES_GFX ? 2 : 6,
```

Comment:
Shouldn't this be 8 instead of 2?
(mainly cross-referencing the vanilla table https://github.com/pret/pokeemerald/blob/e45fe13af4551a080704b6a2a2ee87000c43d2b7/src/data/pokemon_graphics/enemy_mon_elevation.h#L64)

Reply:
That sprite for Deoxys-N wasn't going to show up at all, the 8 was used by its Speed Forme :)
![pokeemerald-0 (2)](https://github.com/user-attachments/assets/0ec8f0b7-653a-48df-b072-8ade5fed4385)
I got the 2 by testing manually, there wasn't much I could refer to for placing the Normal form and that's why I left it alone besides the back that was easily observable (tried to look up Pokeruby's code, but the elevations there looks different from Emerald, and I couldn't find the section regarding the other placements)